### DISCUSSION TITLE
No "tpl_c" directory

### DISCUSSION THREAD

**Discussion by BGARNIER98** [LibreBooking/librebooking#513] [Q&A] [answered]
Hello,
I'm currently in the process of installing your application on an Ubuntu 22.04 server.

I'm at the stage where I would like to automatically install the databases. However, when I open the page "Web/install/", I have this error message : 
"The web server (such as _www on Mac or apache on Linux) must have write access to ../../tpl_c.
The permissions are currently set as 0
You can either change ../../tpl_c to group for example: _www/apache accordingly
Or change ../../tpl_c to have permission 777 which is only for testing due to security vulnerability."

I tried to create an empty directory named "tpl_c", and giving it 777 permissions.

After that, when I refresh the page, I get a white page in my browser.

Would you have a solution ?

Best regards.

**Comment by jmsp867** [comment]
Hi,
Same problem here.

**Comment by treborin** [comment, marked as answer]
Pls. see issue #39 
And for the white paga, pls. check this issue #243
